docs/content/integrations/looker.mdx

```diff
@@ -26,8 +26,6 @@ This guide provides instructions for using Dagster with Looker using the `dagste
 
 </details>
 
-</details>
```

Comment:
Shouldn't we expect some tests to fail if such an error is introduced? What would be the best way to mitigate this problem currently? 

Double-checking the document preview seems like a good option, but my guess is that it's often overlooked for small changes like this.

Reply:
This will be fixed in the new docs-beta site as part of the build process.